radio/src/storage/yaml/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ elseif(PCB STREQUAL NV14)
 elseif(PCB STREQUAL PL18)
   if(PCBREV STREQUAL NB4P)
     set(YAML_GEN_OUTPUT storage/yaml/yaml_datastructs_nb4p.cpp)
+  elseif(PCBREV STREQUAL NV14)
```

Comment:
```suggestion
  elseif(PCBREV STREQUAL RADIO_NV14_FAMILY)
```
Should this be?

Reply:
Same here, I have 2 versions to ensure the old target still works.

Reply:
New version is PCB=PL18 and PCBREV=NV14